react/demo/src/daisyui/samples/toast/Default.route.tsx

```diff
@@ -0,0 +1,27 @@
+import type {ToastApi} from '@agnos-ui/react-headless/components/toast';
+import {useRef} from 'react';
+import {Toast} from './Toast';
+
+const TailwindDemo = () => {
+	const refToast = useRef<ToastApi>(null);
```

Comment:
We discussed last week that we want the most simple DaisyUI examples.
So please remove usage of the api (you can create a react state instead and control the visible prop).